test/living-html/htmlimageelement.js

```diff
@@ -1,11 +1,103 @@
 "use strict";
+const fs = require("fs");
+const path = require("path");
 const jsdom = require("../..");
 
 exports["Image constructor should create a HTMLImageElement with specified width and height"] = t => {
   const window = jsdom.jsdom().defaultView;
   const image = new window.Image(100, 200);
 
-  t.equal(image.width, 100, "width should be set to the passed constructor parameter");
-  t.equal(image.height, 200, "height should be set to the passed constructor parameter");
+  t.strictEqual(image.width, 100, "width should be set to the passed constructor parameter");
+  t.strictEqual(image.height, 200, "height should be set to the passed constructor parameter");
   t.done();
 };
+
+exports["loading image from valid external URL"] = t => {
+  if (!isCanvasInstalled(t)) {
+    return;
+  }
+  const window = jsdom.jsdom("", { features: { FetchExternalResources: ["img"] } }).defaultView;
+  const image = new window.Image();
+  const src = "file://" + path.resolve(__dirname, "files/image.png");
+  t.strictEqual(image.width, 0, "before loading, width should be 0");
+  t.strictEqual(image.height, 0, "before loading, height should be 0");
+  t.strictEqual(image.naturalWidth, 0, "before loading, naturalWidth should be 0");
+  t.strictEqual(image.naturalHeight, 0, "before loading, naturalHeight should be 0");
+  t.strictEqual(image.complete, false, "before loading, complete should be false");
+  t.strictEqual(image.src, "", "before loading, src should be an empty string");
```

Comment:
This is incorrect. `src` should, per spec, reflect the `src=""` content attribute. It should not change in the course of loading.